README.md

```diff
@@ -28,4 +43,6 @@ See [CONTRIBUTING.md](https://github.com/emberjs/ember.js/blob/master/CONTRIBUTI
 
 Cross-browser testing provided by:
 
-<a href="http://browserstack.com"><img height="70" src="https://p3.zdusercontent.com/attachment/1015988/PWfFdN71Aung2evRkIVQuKJpE?token=eyJhbGciOiJkaXIiLCJlbmMiOiJBMTI4Q0JDLUhTMjU2In0..aUrNFb8clSXsFwgw5BUTcg.IJr5piuCen7PmSSBHSrOnqM9K5YZfxX3lvbp-5LCqoKOi4CjjgdA419iqjofs0nLtm26FMURvZ8JRTuKB4iHer6lGu5f8dXHtIkYAHjP5fXDWkl044Yg2mSdrhF6uPy62GdlBYoYxwvgkNrac8nN_In8GY-qOC7bYmlZyJT7tsTZUTYbNMQiXS86YA5LgdCEWzWreMvc3C6cvZtVXIrcVgpkroIhvsTQPm4vQA-Uq6iCbTPA4oX5cpEtMtrlg4jYBnnAE4BTw5UwU_dY83ep5g.7wpc1IKv0rSRGsvqCG_q3g" alt="BrowserStack"></a>
+
+<a href="https://www.browserstack.com/automate/public-build/N0RzTStiOEVmUmFIOHZESHZ0SHc2ZnRtWkdId1VTYjBBNmdUbFdGQjhJMD0tLXlnakFXc2lFVk54YlFJeTZpMElGSGc9PQ==--587563c31781ae54de26452a7d27bd451607b000"><img src='https://www.browserstack.com/automate/badge.svg?badge_key=N0RzTStiOEVmUmFIOHZESHZ0SHc2ZnRtWkdId1VTYjBBNmdUbFdGQjhJMD0tLXlnakFXc2lFVk54YlFJeTZpMElGSGc9PQ==--587563c31781ae54de26452a7d27bd451607b000'/></a>
```

Comment:
This needs to stay the BrowserStack logo per BrowserStack request when giving us the ability to run Cross-browser tests.

Reply:
https://www.browserstack.com/open-source

Reply:
the alt tag also needs to stay on the img.